sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -68,6 +69,45 @@ public static partial class SearchFilter
         public static string Create(System.FormattableString filter) { throw null; }
         public static string Create(System.FormattableString filter, System.IFormatProvider formatProvider) { throw null; }
     }
+    public static partial class SearchIndexingBufferedSenderExtensions
+    {
+        public static void DeleteDocuments(this Azure.Search.Documents.SearchIndexingBufferedSender<Azure.Search.Documents.Models.SearchDocument> indexer, string key, System.Collections.Generic.IEnumerable<string> documentKeys, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { }
+        public static System.Threading.Tasks.Task DeleteDocumentsAsync(this Azure.Search.Documents.SearchIndexingBufferedSender<Azure.Search.Documents.Models.SearchDocument> indexer, string key, System.Collections.Generic.IEnumerable<string> documentKeys, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class SearchIndexingBufferedSenderOptions<T>
+    {
+        public SearchIndexingBufferedSenderOptions() { }
+        public bool AutoFlush { get { throw null; } set { } }
+        public System.TimeSpan? AutoFlushInterval { get { throw null; } set { } }
+        public System.Threading.CancellationToken FlushCancellationToken { get { throw null; } set { } }
+        public System.Func<T, string> KeyFieldAccessor { get { throw null; } set { } }
+    }
+    public partial class SearchIndexingBufferedSender<T> : System.IAsyncDisposable, System.IDisposable
+    {
+        protected internal SearchIndexingBufferedSender(Azure.Search.Documents.SearchClient searchClient, Azure.Search.Documents.SearchIndexingBufferedSenderOptions<T> options = null) { }
```

Comment:
Hmm.  That's a fair point.  I was thinking you could dump out the actual batches that got sent from the mock for interesting reading, but it's more than we need to open the API at this point.  Good catch.

Reply:
The client passed in wouldn't be the mock, though. Great idea if it was, though!

Reply:
...or were you thinking it'd work like so with Moq?
```csharp
var senderMock = new Mock<SearchIndexingBufferedSender>(searchClientMock, ...);
```?
Off hand, I don't know if the protected constructor could be called, but if so, I could see this working. But I don't think it'd be obvious without documenting how it was intended to be used.

Reply:
That's kind of what I was imagining, but I obviously didn't put a ton of thought into it.  :smile:  I'm going with an empty `protected` for now because it's the safest.